frontend/src/MyTeam/MyTeamView.jsx

```diff
@@ -2,14 +2,48 @@ import "./MyTeamView.css";
 import Header from "../Header/Header.jsx";
 import PlayerCardList from "./PlayerCardList.jsx";
 import StatsTable from "../TableComponents/StatsTable";
-import { useEffect, useState } from "react";
+import { useEffect, useState, useContext } from "react";
+import { UserContext } from "../UserContext.js";
 
 function MyTeamView() {
+  const [myTeamPlayers, setMyTeamPlayers] = useState([]);
+  const [playersStats, setPlayersStats] = useState([]);
+  const userContext = useContext(UserContext);
+
+  async function fetchMyTeam() {
+    const queryUrl = new URL(`http://localhost:5000/myTeamPlayers`);
+    queryUrl.searchParams.append("userId", userContext.user.id);
+    const response = await fetch(queryUrl);
+    const data = await response.json();
+    setMyTeamPlayers(data);
+  }
+
+  async function fetchSinglePlayer(playerId) {
+    const queryUrl = new URL(`http://localhost:5000/singlePlayerStats`);
+    queryUrl.searchParams.append("playerId", playerId);
+    const response = await fetch(queryUrl);
+    const player = await response.json();
+    setPlayersStats((playersStats) => [...playersStats, player]);
+  }
+
+  useEffect(() => {
+    fetchMyTeam();
+  }, []);
+
+  useEffect(() => {
+    myTeamPlayers.map((player) => {
+      fetchSinglePlayer(player.playerId);
+    });
+  }, [myTeamPlayers]);
+
   return (
     <div className="myTeamView">
       <Header />
-      <PlayerCardList myTeamStats={[]} />
-      <StatsTable playersList={[]} />
+      <PlayerCardList
+        myTeamPlayers={myTeamPlayers}
+        playersStats={playersStats}
+      />
+      <StatsTable playersList={playersStats} />
```

Comment:
<img width="1718" alt="Screenshot 2024-07-08 at 9 57 14 AM" src="https://github.com/GloriaXM/numbers-game/assets/112780860/0800f2ed-a60d-42a5-af31-a5908af08a59">

Reply:
nit: add margins to player boxes at the top